src/player_activity.h

```diff
@@ -40,6 +40,7 @@ class player_activity
         std::string name;
         std::vector<item_location> targets;
         std::vector<int> values;
+        std::vector<float> f_values;
```

Comment:
If we're going to add floating point values to `player_activity`, I think we should use `double`.

However in this case it looks like you don't really need to store the value and can just recalculate it when you need it.

Reply:
It's removed